platform-sdk/docs/proposals/README.md

```diff
@@ -0,0 +1,150 @@
+# Platform Design Proposal Process
+
+This document describes the kind of code changes that require an accepted design proposal, the process for creating,
+reviewing, and merging platform design proposals, and how accepted proposals can be updated.
+
+The purpose of the design proposal process is to increase the quality of designs for larger and impactful changes
+before time is invested in the code. Higher quality designs reduce misdirected time investments and performance testing
+costs.
+
+The status of all design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Design Proposal Criteria
+
+If code changes fall into one of the categories below, then an accepted design proposal is required before the code can
+merge.
+
+1. Publicly available inputs/outputs/API including disk
+2. Core behaviors
+3. Any non-trivial change that is expected or is likely to impact performance (some judgement must be applied)
+4. Changes to interactions between major systems or teams (platform, services, mirror node, devops, etc.)
+5. Module Structure & Repos
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.drawio.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be created with DrawIO and exported in SVG format. All content
+must be editable (i.e. no images). The only exception is for images that are generated by code, like the wiring diagram.
+These may be drawn on to indicate the intended changes and included as an image.
+
+If any APIs were developed in code as part of the design process, it may be included in the proposal directly or in the
+subdirectory of the proposal, but not merged into the source code of the platform.
```

Comment:
I don't understand this distinction:

> but not merged into the source code of the platform

I assume this means, you can check into the code into the docs/proposals directory, but not into the actual shipping source code.

Reply:
yes, that's what it means